src/hcf-release/jobs/uaa-create-user/templates/run.erb

```diff
@@ -161,7 +177,14 @@ if uaac group get '<%= authority %>' | grep -q '  displayname: <%= username %>'
     status "User %s already in %s" <%= username %> '<%= authority %>'
 elif ! uaac group get '<%= authority %>' | grep -q "    value: ${user_id}" ; then
     status "+ %s" <%= username %>
+    set +e
     uaac member add '<%= authority %>' <%= username %>
+    if [ $? -eq 0 ]; then
+    	status "Add %s to %s successfully" <%= username %> '<%= authority %>'
+    else
+        status "Add %s to %s failed but ignore the error first" <%= username %> '<%= authority %>'
+    fi
+    set -e
```

Comment:
Given that the list of default groups are under our control (see above), would it be possible to check it against that list and skip?